src/core/client/admin/routes/moderate/components/Search/Field.tsx

```diff
@@ -0,0 +1,91 @@
+import cn from "classnames";
+import { Localized } from "fluent-react/compat";
+import React, { FunctionComponent, HTMLAttributes } from "react";
+import { Field as FormField } from "react-final-form";
+
+import { BaseButton, Flex, Icon } from "talk-ui/components";
+
+import styles from "./Field.css";
+
+interface Props extends HTMLAttributes<HTMLInputElement> {
+  /** title of the story */
+  title: string;
+  className?: string;
+  focused?: boolean;
+}
+
+const Field: FunctionComponent<Props> = ({
+  title,
+  focused,
+  className,
+  onBlur,
+  onChange,
+  ...rest
+}) => {
+  const placeholder = focused
+    ? "Use quotation marks around each search term (e.g. “team”, “St. Louis”)"
```

Comment:
Ensure all string literals are translated.

Reply:
This one is actually translated :-)